packages/google_sign_in/android/src/main/java/io/flutter/plugins/googlesignin/GoogleSignInPlugin.java

```diff
@@ -198,12 +206,12 @@ public GoogleSignInAccount getCurrentAccount() {
       return currentAccount;
     }
 
-    private void checkAndSetPendingOperation(String method, Result result) {
+    private void checkAndSetPendingOperation(String method, Result result, Object data) {
```

Comment:
Keep the existing signature and just call `checkAndSetPendingOperation` with null data. That way you don't need to update all callers.